src/components/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  statusFilter: 'all' | 'active' | 'completed';
+  onFilterChange: (filter: 'all' | 'active' | 'completed') => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  statusFilter,
+  onFilterChange,
+}) => {
+  const notCompletedTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${notCompletedTodos.length} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${statusFilter === 'all' ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => onFilterChange('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link ${statusFilter === 'active' ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={() => onFilterChange('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${statusFilter === 'completed' ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method